.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+  "recommendations": [
+    "rvest.vs-code-prettier-eslint"
+  ]
+}
```

Comment:
May need to add line break here?

Reply:
Hah, Prettier was not getting applied to extensions.json itself